scripts/toolbox/modules/TestExecutor.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+
+import tests.RocksDBCacheSizeTest as RocksDBCacheSizeTest
```

Comment:
```suggestion
from tests import RocksDBCacheSizeTest
```

Reply:
This actually works. For whatever reasons, my IDE says this is wrong: 
![image](https://github.com/arangodb/arangodb/assets/1244163/d18d9586-6d28-48e3-9e91-64b4ab115b5d)